docs/source/conf.py

```diff
@@ -45,10 +46,48 @@
     'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
-    'sphinx.ext.mathjax',
     'sphinx.ext.napoleon',
     'sphinx.ext.viewcode',
+    'sphinxcontrib.katex',
+]
+
+# katex (mathjax replacement) macros
+#
+#
+
+katex_macros = r'''
+"\\i": "\\mathrm{i}",
+"\\e": "\\mathrm{e}^{#1}",
+"\\w": "\\omega",
+"\\vec": "\\mathbf{#1}",
+"\\x": "\\vec{x}",
+"\\d": "\\operatorname{d}\\!{}",
+"\\dirac": "\\operatorname{\\delta}\\left(#1\\right)",
+"\\scalarprod": "\\left\\langle#1,#2\\right\\rangle",
+'''
+
+# katex options
+#
+#
+
+katex_options =r'''
+delimiters : [
+   {left: "$$", right: "$$", display: true},
+   {left: "\\(", right: "\\)", display: true},
+   {left: "\\[", right: "\\]", display: true}
 ]
+'''
+
+# katex_macros = r'''
```

Comment:
should these be removed?

Reply:
They don't seem very useful as comments. Are these semantically meaningful? If not, can you remove them?